app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -752,14 +835,13 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
         b.settingsActivityOnDeviceBlockDesc.text = getString(R.string.settings_local_blockList_desc1)
     }
 
-    private fun updateDownloadInitiated(){
+    private fun updateDownloadInitiated() {
         b.settingsActivityOnDeviceBlockDesc.text = getString(R.string.settings_local_blocklist_desc2)
         b.settingsActivityOnDeviceBlockSwitch.visibility = View.GONE
         b.settingsActivityOnDeviceBlockProgress.visibility = View.VISIBLE
         b.settingsActivityOnDeviceBlockRefreshBtn.visibility = View.INVISIBLE
     }
 
-
     private fun showDialogForHTTPProxy(isEnabled: Boolean) {
         if (isEnabled) {
```

Comment:
turn this into a guard statement, return early.

if (!isEnabled) {
   // do stuff
   return
}

Reply:
Made as guard statement.